tests/utils.py

```diff
@@ -168,3 +169,34 @@ def assertTensorEqual(self, *args, **kwargs):  # noqa: N802
 
     def assertTensorAlmostEqual(self, *args, **kwargs):  # noqa: N802
         assert_array_almost_equal(*args, **kwargs)
+
+    def get_h5DS_path(
```

Comment:
What does this do? Can you create a docstring?

Reply:
This creates an HDF5 file inside the temp and returns its path. I will add docstring for the same.